site/static/examples/sample-validating-policy.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: sample-validating-admission-policy
+spec:
+  failurePolicy: Fail
+  matchConstraints:
+    resourceRules:
+    - apiGroups:   ["batch"]
+      apiVersions: ["v1"]
+      operations:  ["CREATE", "UPDATE"]
+      resources:   ["jobs"]
+  validations:
+    - expression: "object.metadata.labels.exists_one(key, key == 'kueue.x-k8s.io/queue-name') && object.metadata.labels['kueue.x-k8s.io/queue-name'] != ''"
```

```suggestion
    - expression: "'kueue.x-k8s.io/queue-name' in object.metadata.labels && object.metadata.labels['kueue.x-k8s.io/queue-name'] != ''"
```
Shorter, and looks equivalent given that an object cannot have more than one value for a key.